src/components/confirmation-dialogs.yml

```diff
@@ -26,6 +26,9 @@ confirmationDialogs:
   deleteRoomInvitationConfirmation:
     en: Do you really want to revoke the room invitation sent to "{invitationEmail}"?
     de: Möchtest du die Raumeinladung für "{invitationEmail}" wirklich widerrufen?
+  deleteRoomMessageConfirmation:
+    en: Do you really want to delete the message posted on "{timestamp}"?
+    de: Möchtest du die Nachricht geschrieben am "{timestamp}" wirklich löschen?
```

Comment:
As we already nicely localize the timestamp with `formatDate(msg.createdOn)` there's no need to use quotes here. Also, in German it would sound more natural to say "Möchtest du die Nachricht vom {timestamp} wirklich löschen?"

Reply:
I wasn't sure due to the comma in the date (for the time separation), thought it might look better in quotes.
Removed quotes here and in a similar case of another confirmation dialog.

And corrected the German version, of course 👍